templates/web/base/reports/_ward-list.html

```diff
@@ -12,7 +12,8 @@ <h2>[% ward_text %]</h2>
       [% FOR child IN children.values.sort('name') %]
         <li class="item-list__item item-list--wards__item">
           <label class="[% show_wards_shown_class %]">
-            <input type="checkbox" name="ward" value="[% child.name | html %]">
+            [% SET value_type = child.name _ "?type=" _ child.type %]
+            <input type="checkbox" name="ward" value="[% value_type | html %]">
```

Comment:
Also no changes to the other_districts-list.html file, so I assume this wouldn't work if you selected the ward with the same name.

Reply:
The params are not really params - perhaps I should use something else like a colon to distinguish them? They are values passed to ward_check in an array rather than a request passed so would need to be changed more fundamentally.

Reply:
Only the firsts checked in order to populate ward_code, and then they are all immediately stripped? So I don't understand your point - why could you not pass the value type in a hidden field into ward_code, rather than reading it off the first ward passed in and then stripping it from all of them?